src/components/productSummary/ProductSummary.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import "./ProductSummary.css";
+
+const ProductSummary = ({
+  accommodation,
+  address,
+  guests,
+  bedrooms,
+  beds,
+  baths,
+  starGrade,
+  reviews,
+}) => {
+  return (
+    <div className="product-summary-container">
+      <h2 className="address">
+        {accommodation} in {address}
+      </h2>
+      <div>
+        <span>
+          {guests} {guests > 1 ? "guests " : "guest "}
+        </span>
+        <span>&#183;</span>
+        <span>
+          {bedrooms} {bedrooms > 1 ? "bedrooms " : "bedroom "}
+        </span>
+        <span>&#183;</span>
+        <span>
+          {beds} {beds > 1 ? "beds " : "bed "}
+        </span>
+        <span>&#183;</span>
+        <span>
+          {baths} {baths > 1 ? "baths " : "bath "}
+        </span>
+      </div>
```

Comment:
I realized this is a list with 4 different variables. A way to refactor the code would be using using .map() function and rendering a list.

Reply:
Thank you for your nice comment. I used a loop to show the information.